packages/wput.rb

```diff
@@ -0,0 +1,28 @@
+require 'package'
+
+class Wput < Package
+  description 'wput is a command line file upload tool, the opposite of wget'
+  homepage 'http://wput.sourceforge.net/'
+  version '0.6.1'
+  source_url 'https://prdownloads.sourceforge.net/wput/wput-0.6.1.tgz'
+  source_sha256 '67125acab1d520e5d2a0429cd9cf7fc379987f30d5bbed0b0e97b92b554fcc13'
+
+  depends_on 'gnutls'
+
+  def self.preinstall
+    system "curl -Ls -o config.guess 'https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'"
+    system "curl -Ls -o config.sub 'https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'"
+  end
+
+  def self.build
+    system './configure',
+           "--prefix=#{CREW_PREFIX}",
+           "--libdir=#{CREW_LIB_PREFIX}"
```

Comment:
Do we need anything like `--disable-maintainer-mode` or `--disable-debugger-checking`?  If those options are not available, no worries.

Reply:
They're not available.